src/platform/silabs/SilabsConfig.h

```diff
@@ -134,6 +134,7 @@ class SilabsConfig
     static constexpr Key kConfigKey_Provision_Request  = SilabsConfigKey(kMatterFactory_KeyBase, 0x28);
     static constexpr Key kConfigKey_Provision_Version  = SilabsConfigKey(kMatterFactory_KeyBase, 0x29);
     static constexpr Key kOtaTlvEncryption_KeyId       = SilabsConfigKey(kMatterFactory_KeyBase, 0x30);
+    static constexpr Key kConfigKey_HardwareVersion    = SilabsConfigKey(kMatterFactory_KeyBase, 0x31);
```

Comment:
There are lot of "gaps" in the factory key ids.
[0x18 to 0x1D]
[0x2A to 0x2F] Lets fills them up before continuing to 0x31